test/integration/controller/jobset_controller_test.go

```diff
@@ -1125,6 +1125,25 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 				},
 			},
 		}),
+		ginkgo.Entry("elastic jobset; scale up replicated jobs", &testCase{
```

Comment:
note: this isn't working yet. Once I figure out validation, I'll come back to this.